less_5/51.py

```diff
@@ -0,0 +1,11 @@
+abs_path = r'C:\Users\pocht\OneDrive\Рабочий стол\blob.txt'
+
+
+def split_path(abs_path: str) -> tuple():

```

Comment:
Вызывать тут функцию – не самая удачная идея
```suggestion
def split_path(abs_path: str) -> tuple:

```